### DISCUSSION TITLE
--bind in fzf_opts don't work

### DISCUSSION THREAD

**Discussion by Grueslayer** [ibhagwan/fzf-lua#1168] [Q&A] [answered]
Hi,

I try to get fzf-lua working as in my vanilla vim but I'm not able to pass a key binding (no action!) to FZF.
I try to map `<Ctrl>-A` to select all the entries:

```
      require("fzf-lua").setup({
        fzf_opts = {
          ['--layout'] = 'default',
          ['--bind'] = 'ctrl-a:select-all'
        },
```
The bind worked like a charm in fzf.vim but does nothing in fzf-lua.

Is this a bug or do I have to enter that in another way?

**Comment by ibhagwan** [comment, marked as answer]
I’m AFK so can’t write it for you but take a look at `keymap.fzf` under the default options (in the main README).

**Reply by Grueslayer** [reply]
Okay.... you've put some Alt-q(Alt-l)/Alt-a already in your default, I previously mapped those to Ctrl-q/-a, maybe I take that.

**Reply by ibhagwan** [reply]
I found it less conflicting with our common binds other users have (window manager, term, etc).
